### PR TITLE
SN DSA Release

### PR DESCRIPTION
**Pull request by sarahgonicholson**
In `commands/release-file.py` and `commands/create-annotated-filenames.py`:
  * Assay and sequencer codes value set to XX for DSA fasta files and chain files
 * For Supplementary Files, use `haplotype`, `target_assembly`, and `source_assembly` properties to create annotated filenames for chain and fasta files

### CODE REVIEW

**Comment by alexander-veit** [comment]
The tests are failing. I think it's related to your latest change

File: src/encoded/commands/create_annotated_filenames.py [hunk 1]
```diff
@@ -816,7 +824,9 @@ def get_analysis(
         software_and_versions, reference_genome_code
     )
     if file_format_utils.is_chain_file(file_extension):
-        value = f"{value}{ANALYSIS_INFO_SEPARATOR}{get_chain_file_value(file)}"
+        value = ANALYSIS_INFO_SEPARATOR.join([value,get_chain_file_value(file)]) if value else get_chain_file_value(file)
+    if file_format_utils.is_fasta_file(file_extension):
```
Comment (alexander-veit) [inline src/encoded/commands/create_annotated_filenames.py:828]: Not quite sure I understand. You are only testing for fasta file here. Do you always want to append`dsa` in that case?

File: src/encoded/commands/release_file.py [hunk 2]
```diff
@@ -354,17 +358,16 @@ def add_file_patchdict(self, dataset: str) -> None:
...
         patch_body = {
             item_constants.UUID: item_utils.get_uuid(self.file),
             item_constants.STATUS: item_constants.STATUS_RELEASED,
-            file_constants.FILE_SETS: file_set_accessions,
             file_constants.DATASET: dataset,
             file_constants.ACCESS_STATUS: access_status,
             file_constants.ANNOTATED_FILENAME: annotated_filename_info.filename,
         }
-
+        if not supp_file_utils.is_reference_conversion(self.file) and not supp_file_utils.is_reference_genome(self.file):
```
Comment (alexander-veit) [inline src/encoded/commands/release_file.py:369]: You are making this check already in `get_file_sets_from_file`. Should we just test here if `file_set_accessions` is empty?

**Review by alexander-veit — COMMENTED** [review]
Generally looks good. Some small clarifications needed. Also test issue must be adressed.

File: src/encoded/commands/create_annotated_filenames.py (hunk 1 above)
Comment (sarahgonicholson) [reply src/encoded/commands/create_annotated_filenames.py:828]: Great point. I'll add a specification for only DSA fasta files

**Review by alexander-veit — APPROVED** [review]

**Pull request merged**